### DISCUSSION TITLE
Final report always has an error when I run in docker

### DISCUSSION THREAD

**Discussion by huang-kaixin** [The-OpenROAD-Project/OpenROAD-flow-scripts#2602] [Q&A]
![image](https://github.com/user-attachments/assets/bdec40da-84ae-4b80-9b36-375d791088a0)
I ran ibex design and JPEG design in Docker and ended up with the above error, unable to generate 6_report.log. What is the problem and how to fix it? Thanks

**Comment by maliberty** [comment]
What OS are you using?

**Reply by huang-kaixin** [reply]
I created it from https://hubgw.docker.com/r/openroad/flow-ubuntu 22.04-builder

**Comment by maliberty** [comment]
@vvbandeira any thoughts?

**Comment by vvbandeira** [comment]
@huang-kaixin 
How are you starting the docker container? 
What is your Docker version?
What is your base OS?
